<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601916583963' stop='1601916635833'>
    <name>Add KNOS Relationship</name>
    <title>Add KNOS Relationship</title>
    <test-cases>
        <test-case start='1601916583963' status='passed' stop='1601916624881'>
            <name>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</name>
            <title>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='b2b4f33c-86ae-40b2-9d8d-84903d117458-attachment.png' type='image/png' size='49045'/>
            </attachments>
        </test-case>
        <test-case start='1601916624881' status='passed' stop='1601916629074'>
            <name>AddNewKNOS- It should add new casetype and subtype releationship to system!</name>
            <title>AddNewKNOS- It should add new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='cdb898fd-55c9-479a-8557-fd54d71129e4-attachment.png' type='image/png' size='44290'/>
            </attachments>
        </test-case>
        <test-case start='1601916629074' status='failed' stop='1601916635833'>
            <name>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</name>
            <title>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='c7d3ae0c-b52e-4f38-b93e-069ca69e299f-attachment.png' type='image/png' size='57898'/>
            </attachments>
            <failure>
                <message>Expected 'Professional Tort' to equal 'CaseType123'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (C:\Users\UX013810\workspace\WCWRUI\Tests\Admin\UC_AD_16\AddNewKNOSTest.js:71:54)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2974:25
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>